src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -157,13 +159,19 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
 
   const auto* thread_state_slice =
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
+  auto begin_ns = thread_state_slice->begin_timestamp_ns();
+  auto end_ns = thread_state_slice->end_timestamp_ns();
```

Comment:
Nit: We are trying to avoid `auto` when the type is not apparent. From the style guide:

```
The fundamental rule is: use type deduction only to make the code clearer or safer, and do not use it merely to avoid the inconvenience of writing an explicit type. When judging whether the code is clearer, keep in mind that your readers are not necessarily on your team, or familiar with your project, so types that you and your reviewer experience as unnecessary clutter will very often provide useful information to others. For example, you can assume that the return type of make_unique<Foo>() is obvious, but the return type of MyWidgetFactory() probably isn't.
```

Source: go/cstyle#Type_deduction

Reply:
Yeah, forgot to change this. In the previous version it was alright. 

It should be okay now, please take a look.